Assets/Scripts/Gameplay/Player/PlayerMovement.cs

```diff
@@ -21,6 +21,7 @@ void Start()
         anim = GetComponent<Animator>();
         body = GetComponent<Rigidbody2D>();
         playerStats = GetComponent<PlayerStats>();
+        spriteRenderer = GetComponent<SpriteRenderer>();
```

Comment:
you should have a requirescomponent above the class for all of these or null check them all here for futureproofing.